source/utilities.tex

```diff
@@ -12224,30 +12224,24 @@
 The following definitions apply to this Clause:
 
 \pnum
-\indexdefn{call signature}%
-A \term{call signature} is the name of a return type followed by a
+A \defn{call signature} is the name of a return type followed by a
 parenthesized comma-separated list of zero or more argument types.
 
```

Comment:
For this (and those that follow), the index entries should probably use tilde instead of space.